plugins/ImageGraph/ImageGraph.php

```diff
@@ -174,4 +174,10 @@ public function getReportMetadata(&$reports, $info)
             }
         }
     }
+
+    public static function getDefaultGraphEvolutionLastPeriods()
+    {
+        $lastPeriods = Config::getInstance()->General['graphs_default_evolution_graph_last_days_amount'];
```

Comment:
should we maybe fail if someone configures a value <= 0? so the user can see it was not correctly changed? not needed though...